modules/ROOT/partials/configuration/valid_elements.adoc

```diff
@@ -60,6 +61,7 @@ If you just want to add or change some behavior for a few items, use the link:{b
 
 Wildcards such as `*`,`+`,`?` may be used in element or attribute name matching.
 
+[discrete]
 === Variables
 
 |===
```

Comment:
This table has some rendering issues, as the name takes up a lot of space. We should be adding the `cols` markup to specify the first column should take up less space than the last. See https://docs.asciidoctor.org/asciidoc/latest/tables/add-columns/